dxp/liferay_global_event_handler.md

```diff
@@ -0,0 +1,213 @@
+# Liferay’s Global Event Handler
```

Comment:
Technically, I think "event bus" is the more commonly used term to describe the entire event system. "Handler" just refers to a specific function that is notified of an event when it happens.

Separately, I see "smart quotes" in this file. Truly evil, just sayin'.

![true-evil](https://user-images.githubusercontent.com/7074/92577419-e6371680-f28a-11ea-9ddc-5642d62ed7ab.gif)

Reply:
👀

Reply:
> I think "event bus" is the more commonly used term to describe the entire event system. "Handler" just refers to a specific function that is notified of an event when it happens.

Yes, "handler" is just a part of it. I'll change :)

Reply:
When searching about DXP "event bus" I found some links that mentions "IPC(Inter Portlet Communication)" instead. Should I update and use Event Bus or IPC? What do you think? @wincent @jbalsas @david-truong

See:
- http://www.liferaysavvy.com/2014/01/liferay-client-side-inter-portlet.html
- https://liferay.dev/forums/-/message_boards/message/38984904

Reply:
IPC is something completely different, and DXP has indeed a Message (Event) Bus but that refers to a backend event bus. 

This is meant for internal consumption so we shouldn't need to get into small distinctions like that. Our target audience (internal liferay frontend developers) won't be confused by those terms (if they even know what they mean).

Reply:
it was turned: Liferay’s Global Event Bus 🎉